src/main/webapp/WEB-INF/view/profile.jsp

```diff
@@ -13,20 +13,21 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 --%>
-<%@ page import="codeu.model.data.User" %>
 <%@ page import="java.util.List" %>
-<%@ page import="codeu.model.data.Conversation" %>
+<%@ page import="java.time.Instant" %>
+<%@ page import="codeu.model.data.User" %>
 <%@ page import="codeu.model.data.Message" %>
+<%@ page import="codeu.model.data.Conversation" %>
 <%@ page import="codeu.model.store.basic.UserStore" %>
 <%@ page import="codeu.model.store.basic.MessageStore" %>
 <%@ page import="codeu.model.data.StyleText" %>
 <%@ page import="codeu.model.util.Util" %>
-<%@ page import="java.time.Instant" %>
 
 <%
-User user = (User) request.getAttribute("user");
+User active_user = (User) request.getAttribute("user");
```

Comment:
Variable names in Java should always be in camelCase, so use `activeUser` and `profileOwner`.